charts/common/templates/pdb.yaml

```diff
@@ -0,0 +1,21 @@
+{{- if .Values.podDisruptionBudget.enabled }}
+apiVersion: policy/v1
+kind: PodDisruptionBudget
+metadata:
+  name: {{ include "common.name" . }}
+  labels:
+    {{- include "common.labels" . | nindent 4 }}
+spec:
+{{- if and (not .Values.podDisruptionBudget.minAvailable) (not .Values.podDisruptionBudget.maxUnavailable) }}
+  minAvailable: "13%"
```

Comment:
maybe this one should be 15% ? according to the comment in `values.yaml`

Reply:
good catch.  Yes, it should be `15%`.  Changed it to test and didn't revert it.